modules/claims_api/app/models/claims_api/auto_established_claim.rb

```diff
@@ -8,6 +8,7 @@ class AutoEstablishedClaim < ApplicationRecord
     attr_encrypted(:form_data, key: Settings.db_encryption_key, marshal: true, marshaler: JsonMarshal::Marshaller)
     attr_encrypted(:auth_headers, key: Settings.db_encryption_key, marshal: true, marshaler: JsonMarshal::Marshaller)
     attr_encrypted(:evss_response, key: Settings.db_encryption_key, marshal: true, marshaler: JsonMarshal::Marshaller)
+    attribute :flashes
```

Comment:
Ok, looks like this line is causing your problem regarding this error:
```
*** TypeError Exception: can't cast Array
```
Just need to remove it.

I commented out the flashes setter in the create and tossed a debug point down right after the create in the controller. That's when I noticed `auto_claim.flashes` was returning `"{ }"` instead of an array as expected. I took a look at the "signature_errors" field on the power of attorney model since it was pretty much identical to what we are trying to achieve here and confirmed that `"{ }"` was not what we should be seeing. Doublechecked the power of attorney model and sure enough it didnt have an attribute line for signature_errors.

Hope that helps!